storage-node/packages/cli/src/commands/consts.ts

```diff
@@ -0,0 +1,2 @@
+// Defines maximum content length for the assets (files)
```

Comment:
Did you consider placing this constant in the existing base.ts file? Both usage of this constant use base.ts.

Reply:
Yes that does look like a better place for it :+1:

Reply:
done in 543f46c4a98eda323091f2a0053ddbc7bc62db00